------------------------------------------------------------------------
r44530 | karl | 2017-06-08 19:16:15 -0400 (Thu, 08 Jun 2017) | 3 lines

luatex.c (luatex_date_info): bump for TL17 update.
luatex-core.lua: disable io.saved_open for TL17 update.


Index: Build/source/texk/web2c/luatexdir/lua/luatex-core.lua
===================================================================
--- Build/source/texk/web2c/luatexdir/lua/luatex-core.lua	(revision 44529)
+++ Build/source/texk/web2c/luatexdir/lua/luatex-core.lua	(revision 44530)
@@ -29,7 +29,7 @@
 local kpseused            = status.kpse_used    -- 0 1
 
 io.saved_open             = io_open  -- can be protected
-io.saved_popen            = io_popen -- can be protected
+-- (deleted for tl17 rebuild) io.saved_popen            = io_popen -- can be protected
 io.saved_lines            = io_lines -- always readonly
 mt.saved_lines            = mt_lines -- always readonly
 
Index: Build/source/texk/web2c/luatexdir/luatex.c
===================================================================
--- Build/source/texk/web2c/luatexdir/luatex.c	(revision 44529)
+++ Build/source/texk/web2c/luatexdir/luatex.c	(revision 44530)
@@ -28,9 +28,11 @@
 
 #define TeX
 
+/* for tl17 update, change luatex_date_info but nothing else,
+   as context depends on the numeric version number. */
 int luatex_version = 100;        /* \.{\\luatexversion}  */
 int luatex_revision = '4';      /* \.{\\luatexrevision}  */
-int luatex_date_info = 2017033109;     /* the compile date is now hardwired :YEAR MONTH DAY HOUR*/
+int luatex_date_info = 2017060901;     /* the compile date is now hardwired :YEAR MONTH DAY HOUR*/
 const char *luatex_version_string = "1.0.4";
 const char *engine_name = my_name;     /* the name of this engine */
 

------------------------------------------------------------------------
